src/index.js

```diff
@@ -1,3 +1,3 @@
 /* @flow */
 
-export { Rifm } from './Rifm';
+export { Rifm } from './rifm';
```

Comment:
What this for?

Reply:
`.flowconfig` modules resolution to work
I added src to modules resultion so `import { Rifm } from 'rifm'` could work only iif name inside src is `rifm` not `Rifm`

Reply:
You are right, switched to `module.name_mapper='^rifm$' -> '../../src'`